arviz/plots/backends/matplotlib/ppplot.py

```diff
@@ -0,0 +1,102 @@
+"""Matplotib Posterior predictive plot."""
+import matplotlib.pyplot as plt
+import matplotlib
+import numpy as np
+
+from . import backend_show
+from ...kdeplot import plot_kde
+from ...plot_utils import _create_axes_grid
+
+colors = {
+    "prior": "red",
+    "posterior": "blue",
+    "observed": "yellow",
+}
+
+
+def plot_pp(
+    ax,
+    length_plotters,
+    rows,
+    cols,
+    figsize,
+    pp_plotters,
+    linewidth,
+    legend,
+    groups,
+    fill_kwargs,
+    plot_kwargs,
+    backend_kwargs,
+    show,
+):
+    """Matplotlib pp plot."""
+    if ax is None:
+        fig, ax = _create_axes_grid(
+            rows * cols, rows, cols, figsize=figsize, backend_kwargs=backend_kwargs
+        )
+        axes = []
+        for i in range(cols, rows * cols, 2 * cols):
+            axes += list(ax[i - cols : i])
+            gs = ax[i].get_gridspec()
+            for j in range(i, i + cols):
+                ax[j].remove()
+            axbig = fig.add_subplot(gs[i // cols, :])
+            axes.append(axbig)
+
+    else:
+        axes = np.ravel(ax)
+        if len(axes) != length_plotters:
+            raise ValueError(
+                "Found {} variables to plot but {} axes instances. They must be equal.".format(
+                    length_plotters, len(axes)
+                )
+            )
+
+    matplotlib.rc("xtick", labelsize=7)
+    matplotlib.rc("ytick", labelsize=7)
```

Comment:
I think `scale_fig_size` should be used here to get textsize, labelsize and linewidth: https://github.com/arviz-devs/arviz/blob/master/arviz/plots/plot_utils.py#L31

Reply:
It's already set in kdeplot. So, I am just removing them.